packages/terra-arrange/src/Arrange.jsx

```diff
@@ -47,21 +55,25 @@ const Arrange = ({
   alignFitStart,
   alignFill,
   alignFitEnd,
+  fitStartMaxWidth,
+  fitEndMaxWidth,
   ...customProps
   }) => {
   if (fitStart === undefined && fitEnd === undefined) {
     throw new Error('At least one of the props: [fitStart, fitEnd] should be supplied.');
   }
+  const fitStartProps = (fitStartMaxWidth !== undefined) ? { maxWidth: fitStartMaxWidth } : { maxWidth: '40%' };
```

Comment:
Instead of doing this you can use default props to set the default for these to '40%'

Reply:
+1, forgot about the defaultProps, updated now